internal/schedule_client.go

```diff
@@ -416,12 +416,31 @@ type (
 		// Memo - Non-indexed user supplied information.
 		Memo *commonpb.Memo
 
-		// SearchAttributes - Indexed info that can be used in query of List schedules APIs. The key and value type must be registered on Temporal server side.
-		// Use GetSearchAttributes API to get valid key and corresponding value type.
+		// SearchAttributes - Specifies Search Attributes that will be attached to the schedule. Search Attributes
+		// are additional indexed information attributed to the schedule and used for search and visibility. The key and
+		// its value type must be registered on Temporal server side.
 		// For supported operations on different server versions see [Visibility].
 		//
+		// nil: leave any pre-existing assigned search attributes intact
+		// empty: remove any and all pre-existing assigned search attributes
+		// attributes present: replace any and all pre-existing assigned search attributes with the defined search
+		//                     attributes, i.e. upsert
+		//
 		// [Visibility]: https://docs.temporal.io/visibility
 		SearchAttributes *commonpb.SearchAttributes
+
+		// TypedSearchAttributes - Specifies Search Attributes that will be attached to the schedule. Search Attributes
```

Comment:
This doc discusses `ScheduleDescription` as if it's used for creating/updating schedules, but it's read only information.